tools/__tasks__/validate-head/javascript-fix.js

```diff
@@ -0,0 +1,23 @@
+const execa = require('execa');
+const getChangedFiles = require('../lib/get-changed-files');
+
+module.exports = {
+    description: 'Fix committed linting errors',
```

Comment:
i guess this is fixing current versions of committed files, should it prompt maybe to check that's ok? if this isn't a hook anymore, is worth checking if uncommitted changes are going to be linted?